src/reynir_correct/pattern.py

```diff
@@ -1413,6 +1736,174 @@ def create_patterns(cls) -> None:
                     None,
                 )
             )
+            # Catch ""
+            cls.add_pattern(
+                (
+                    "vitni",  # Trigger lemma for this pattern
+                    "VP > { VP > 'verða' NP-PRD > { 'vitni' PP > { P > { 'af' } } } }",
```

Comment:
Might want `"VP > { VP > 'verða' .* NP-PRD > { 'vitni' PP > { P > { 'af' } } } }"` to cover cases such as "hann varð því miður vitni af slysinu", i.e. using the wildcard .* to allow intermediate (in this case probably ADVP) nonterminals.

Reply:
Yes, mea culpa - I forgot the .* isn't used inside { } brackets (set operator), only inside [ ] brackets (list operator).